packages/core/src/2d/sprite/SpriteMask.ts

```diff
@@ -200,6 +201,13 @@ export class SpriteMask extends Renderer {
     target.sprite = this._sprite;
   }
 
+  /**
+   * @inheritdoc
+   */
+  override _isCulledByCamera(camera: Camera): boolean {
+    return false;
+  }
```

Comment:
Does this function only return false?  I feel like someone might criticize for this.

Reply:
> Does this function only return false? I feel like someone might criticize for this.

Mask use in scene, so, all ways return false